frontend/packages/console-app/src/components/quick-starts/DrawerComponent.tsx

```diff
@@ -24,17 +24,17 @@ type DrawerComponentProps = {
 
 const DrawerComponent: React.FC<DrawerComponentProps> = ({
```

Comment:
While we're renaming. `DrawerComponent.tsx` => `QuickStartDrawerComponent.tsx`

Reply:
updated